runtime-modules/content/src/benchmarks/benchmarking.rs

```diff
@@ -0,0 +1,226 @@
+
+    create_channel {
+
+        let a in 1 .. T::MaxNumberOfCollaboratorsPerChannel::get(); //max colaborators
+
+        let b in (T::StorageBucketsPerBagValueConstraint::get().min as u32) ..
+            (T::StorageBucketsPerBagValueConstraint::get().max() as u32);
+
+        let c in (T::DistributionBucketsPerBagValueConstraint::get().min as u32) ..
+            (T::DistributionBucketsPerBagValueConstraint::get().max() as u32);
+
+        let d in 1 .. T::MaxNumberOfAssetsPerChannel::get(); //max objs number
+
+        let max_obj_size: u64 = T::MaxDataObjectSize::get();
+
+        let storage_wg_lead_account_id = insert_storage_leader::<T>();
+
+        let distribution_wg_lead_account_id = insert_distribution_leader::<T>();
+
+        let (channel_owner_account_id, channel_owner_member_id) =
+            member_funded_account::<T>(DEFAULT_MEMBER_ID);
+
+        let sender = RawOrigin::Signed(channel_owner_account_id);
+
+        let channel_owner = ChannelOwner::Member(channel_owner_member_id);
+
+        let params = generate_channel_creation_params::<T>(
+            storage_wg_lead_account_id,
+            distribution_wg_lead_account_id,
+            a, b, c, d,
+            max_obj_size,
+        );
+
+    }: _ (sender, channel_owner, params)
+    verify {
+        let channel_id: T::ChannelId = One::one();
+        assert!(ChannelById::<T>::contains_key(&channel_id));
+        // channel counter increased
+        assert_eq!(
+            Pallet::<T>::next_channel_id(),
+            channel_id.saturating_add(One::one())
+        );
+    }
+    /*
+    ===============================================================================================
+    ======================================== CURATOR GROUPS =======================================
+    ===============================================================================================
+    */
+
+    create_curator_group {
+        let a in 0 .. (T::MaxKeysPerCuratorGroupPermissionsByLevelMap::get() as u32);
+
+        let lead_account = insert_content_leader::<T>();
+        let group_id = Pallet::<T>::next_curator_group_id();
+        let permissions_by_level: ModerationPermissionsByLevel::<T> = (0..a).map(
+            |i| (i.saturated_into(), worst_case_content_moderation_actions_set())
+        ).collect();
+    }: _ (
+        RawOrigin::Signed(lead_account),
+        true,
+        permissions_by_level.clone()
+    )
+    verify {
+        let group = Pallet::<T>::curator_group_by_id(group_id);
+        assert!(group == CuratorGroup::create(true, &permissions_by_level));
+        assert_last_event::<T>(Event::<T>::CuratorGroupCreated(group_id).into());
+    }
+
+    update_curator_group_permissions {
+        let a in 0 .. (T::MaxKeysPerCuratorGroupPermissionsByLevelMap::get() as u32);
+
+        let lead_account = insert_content_leader::<T>();
+        let group_id = setup_worst_case_curator_group_with_curators::<T>(
+            T::MaxNumberOfCuratorsPerGroup::get()
```

Comment:
I think It's better to add this, as you cannot create more curators than the min of these two limits (workers and curators), this can differ later in runtime and fail the test.
```suggestion
min(<T as working_group::Config<ContentWorkingGroupInstance>>::MaxWorkerNumberLimit::get(),
        T::MaxNumberOfCuratorsPerGroup::get()),
```